.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/rust:1.36.0
+
+    steps:
+      - checkout
+
+      - run:
+          name: setup
+          command: |
+            rustup install nightly
```

Comment:
I see so we avoid the image's Rust version and instead simply use our own installed through `rustup`. Is that correct?

Reply:
@Robbepop Yep, we'll need some other toolchain instead of the one in docker image. For instance, only `rustfmt-nighly` could do license template check.